mods/binoculars/init.lua

```diff
@@ -59,6 +59,15 @@ minetest.register_craftitem("binoculars:binoculars", {
 	stack_max = 1,
 
 	on_use = function(itemstack, user, pointed_thing)
+		if pointed_thing then
```

Comment:
`pointed_thing` is always specified. Instead, check whether the player used the binoculars on a node: `pointed_thing.type == "node"`.

Reply:
Ok.

Reply:
However we're waiting to see if this PR is actually needed, see issues.